projects/igniteui-angular/src/lib/directives/drag-drop/drag-drop.directive.ts

```diff
@@ -151,52 +157,104 @@ export interface IDragStartEventArgs extends IDragBaseEventArgs {
     cancel: boolean;
 }
 
+export interface IDragLocation {
+    pageX: number;
+    pageY: number;
+}
+
+export interface IDragCustomAnimationArgs {
```

Comment:
I can't see the usage of this interface `IDragCustomAnimationArgs`. The spec states that it should used by `animateToOrigin` and `animateTo`, but those methods are not implemented.